hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -43,26 +41,26 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-You can now boot from an USB mass storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+, CM3+.
+You can now boot from a USB mass storage device in the same way as booting from an SD card - see the following section for details.
 
-## Raspberry Pi 3B+, CM3+
+## Raspberry Pi 3B+, Compute Module 3+
 
-The Raspberry Pi 3B+ and CM3+ support USB mass storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+The Raspberry Pi 3B+ and Compute Module 3+ support USB mass storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
```

Comment:
"settings specific" -> "steps specific" ? (since the sentence ends with "executed")